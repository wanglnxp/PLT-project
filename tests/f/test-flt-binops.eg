int main(){
	float x=6.11;
	float y=3.45;
	print(x+y);
	print(x-y);
	print(x*y);
	print(x/y);
	print(x+y);
	print(x==y);
	print(x<=y);
	print(x>=y);
	print(x<y);
	print(x>y);
	print(x!=y);
	return 0;
}
