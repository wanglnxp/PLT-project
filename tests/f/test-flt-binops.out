9.560000
2.660000
21.079500
1.771014
9.560000
false
false
true
false
true
true
